requirements.txt

```diff
@@ -2,6 +2,7 @@ adal==0.4.3
 applicationinsights==0.10.0
 argcomplete==1.3.0
 colorama==0.3.7
+coverage==4.2
```

Comment:
Does this file only apply to dev-setup or all installation methods? It would seem that if someone did a curl install of the CLI for production use, then they wouldn't need (or probably want) packages related strictly to testing and development.